prowler/config/config.yaml

```
@@ -99,7 +99,9 @@ aws:
       "nodejs10.x",
       "nodejs12.x",
       "nodejs14.x",
+      "nodejs16.x",
       "dotnet5.0",
+      "dotnet7.0",
```

```suggestion
      "dotnet7",
```
The identifier for AWS is like that.